3rdparty/wigner/wigxjpf/CMakeLists.txt

```diff
@@ -47,34 +47,12 @@ check_c_source_compiles("
         __uint128_t b;
     }
     "
-    HAS_UINT128
+    MULTI_WORD_INT_SIZEOF_ITEM
```

Comment:
Why not make this follow the `WIGXJPF_HAVE_` pattern?

Reply:
That would be a question to the wigner developers because that's the name of the constant they use in their code.

Reply:
Ah, right, it is a one-to-one translation.  Oh well, then this is good.